test/unit/gem_download_test.rb

```diff
@@ -114,12 +114,12 @@ def test_update_the_count_atomically
     @rubygem_3 = create(:rubygem)
     @version_4 = create(:version, rubygem: @rubygem_3)
 
-    GemDownload.increment(@version_1.full_name)
-    GemDownload.increment(@version_2.full_name)
-    GemDownload.increment(@version_3.full_name)
-    GemDownload.increment(@version_1.full_name)
-    3.times { GemDownload.increment(@version_3.full_name) }
-    2.times { GemDownload.increment(@version_2.full_name) }
+    GemDownload.increment(1, rubygem_id: @rubygem_1, version_id: @version_1.id)
+    GemDownload.increment(1, rubygem_id: @rubygem_1, version_id: @version_2.id)
+    GemDownload.increment(1, rubygem_id: @rubygem_2, version_id: @version_3.id)
+    GemDownload.increment(1, rubygem_id: @rubygem_3, version_id: @version_1.id)
+    3.times { GemDownload.increment(1, rubygem_id: @rubygem_2, version_id: @version_3.id) }
+    2.times { GemDownload.increment(1, rubygem_id: @rubygem_1, version_id: @version_2.id) }
```

Comment:
The method signature for these methods had changed. This test is skipped but still seemed better to fix it.